content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,257 @@
+implementation that follow the [concept map](https://www.hl7.org/fhir/conceptmap.html)
+specification defined by HL7. The specification maps proprietary codes
+used in client systems with standardized [LOINC codes](https://www.hl7.org/fhir/loinc.html).
+In some cases, these mappings do not cover all the possible variances in
+the ways hospitals have implemented their EHRs. When EHRs were
+originally implemented, it was common to use proprietary codes to
+document labs and measurements that don’t map directly to the LOINC
+codes defined by the FHIR specification.
+
+For the [Observation resource](http://fhir.cerner.com/millennium/dstu2/diagnostic/observation/),
+we have seen apps developed that use a set of LOINC codes as a filter to
+get specific results. If your app uses specific codes to query for data,
+you should determine if the request should be broader and include more
+possible codes to account for different ways data might be coded or work
+with us to determine the best way to return the information you need.
+
+For example, lead can be performed on blood, capillary blood, or venous blood.
+
+- **Hospital A** uses a proprietary code for lead which maps to LOINC code `5671-3` - `(Lead [Mass/volume] in Blood)`
+- **Hospital B** uses a proprietary code for lead which maps to LOINC code `77307-71` - `(Lead [Mass/volume] in Venous blood)`
+
+If you are using a specific code to query for lead and use code
+`5671-3`, then your app would only return results when installed at
+**Hospital A**.  Your app would return nothing if installed at
+**Hospital B**. You need to ensure that all codes are accounted for in
+order for the app to work at all sites.
+
+Another scenario is where hospitals have different tests and codes setup
+for different methodologies. An example is when a separate test and code
+is created to document blood lead collected by venipuncture versus a
+capillary draw, which is a common collection method for children. On
+occasion, client proprietary codes may not map directly to any LOINC
+code. For special cases where we discover a proprietary code that does
+not map cleanly to any standard, we will work with you to figure out the
+best option for that mapping during deployment.
+
+For each of these scenarios, it is one thing to validate against our
+sandbox to adhere to the FHIR specification, but we also need to
+validate your app at each client site to ensure that there aren’t any
+hospital-specific considerations to be aware of. We will do this when we
+connect your app to a hospital domain. We will work with you to resolve
+any of these hospital-specific considerations.
+
+The above examples use lead, but similar considerations could exist for
+virtually any lab or measurement where concept mapping and specific
+LOINC codes are used.  If your app uses specific LOINC codes to query
+for information, reach out to us and we will help you work through any
+considerations that might exist.
```

Comment:
Should we mention of any preferred way of communication with Cerner to address any such loinc mapping considerations ?

Reply:
linked to google group - 062ac1e0d6134f01dfa35bed38b5a3f080e63084

This should be our preferred method of communication.